packages/eslint-plugin/tests/rules/no-floating-promises.test.ts

```diff
@@ -191,12 +184,8 @@ async function test() {
     () => {},
```

Comment:
[Non-actionable] Aside: the `promiseValue.finally(() => {});` on line 173 (unchanged by this PR) can be removed because that test case doesn't define a `promiseValue` value. It's an existing typo 😄. Not requesting any action, just noting for reference. This is good `good first issue` fodder.